src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -712,6 +697,56 @@ public Properties getConsumerProperties() {
     return consumerProps;
   }
 
+  public Properties getAdminProperties() {
+    Properties consumerProps = new Properties();
+    //copy cover the properties with prefixes "client." and  "admin."
+    addPropertiesWithPrefix("client.", consumerProps);
+    addPropertiesWithPrefix("admin.", consumerProps);
+    return consumerProps;
+  }
+
+  public String bootstrapBrokers() {
```

Comment:
ok, so now I see this is for backwards compatibility and was previously in `ProducerPool` (which was not a good place for it). It's a little bit questionable putting this in `KafkaRestConfig` too, but better than `ProducerPool`.